app/controllers/additional_submission_informations_controller.rb

```diff
@@ -0,0 +1,41 @@
+# Voluntary submission information controller for creation only.
+class AdditionalSubmissionInformationsController < ApplicationController
+  def new
+    @info = AdditionalSubmissionInformation.new(ori: find_ori)
+  end
+
+  def create
+    info_params = get_formatted_params(params, AdditionalSubmissionInformation)
+    @info = AdditionalSubmissionInformation.new(info_params)
+    @info.ori = find_ori
+
+    if @info.save
+      redirect_user
+    else
+      render :new
+    end
+  end
+
+  private
+
+    def find_ori
+      if @current_user.allowed_oris.include?(params[:ori])
+        params[:ori]
+      else
+        @current_user.ori
```

Comment:
Shouldn't we raise an error in this case? The user is somehow trying to post information for a disallowed ORI. It seems strange to me to just quietly change it to their own ORI?